apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/files/shards/ShardPresentData.java

```diff
@@ -6,14 +6,26 @@
 
 import java.util.List;
 import lombok.AllArgsConstructor;
-import lombok.Data;
+import lombok.Getter;
 import lombok.NoArgsConstructor;
+import lombok.Setter;
 
 @AllArgsConstructor
 @NoArgsConstructor
-@Data
+@Setter
+@Getter
 public class ShardPresentData {
     public Long shardId;
     public String shardFileId; // contains shardId + chainId in special format
     public List<String> additionalFileIDs;
+
+    @Override
+    public String toString() {
```

Comment:
please, consider lombok's @ToString here

Reply:
done, reverted to @Data annotation